keyboards/lily58/keymaps/chuan/keymap.c

```diff
@@ -0,0 +1,240 @@
+#include QMK_KEYBOARD_H
+
+#ifdef PROTOCOL_LUFA
+  #include "lufa.h"
+  #include "split_util.h"
+#endif
+#ifdef SSD1306OLED
+  #include "ssd1306.h"
+#endif
+
+
+extern keymap_config_t keymap_config;
+
+#ifdef RGBLIGHT_ENABLE
+//Following line allows macro to read current RGB settings
+extern rgblight_config_t rgblight_config;
+#endif
```

Comment:
This may not be needed, if you use the rgblight "get" functions. 

https://docs.qmk.fm/#/feature_rgblight?id=query

Reply:
I don't have RGB on my keyboard so I won't be able to test this before merging into `future`. I'd leave this for future work since it's not a breaking change in and of itself.